spec/lib/proxmox_spec.rb

```diff
@@ -33,53 +33,55 @@
 
   describe 'get' do
```

Comment:
generally i use the describe blocks to indicate which method of a class is being tested.. and use the it blocks to describe the indiviual test.. this way there is a 1:1 correlation between class methods and describe blocks.. to name it anything but 'get' would break that correlation